app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -45,7 +45,10 @@ public Result getSelfParticipant() {
         
         StudyParticipant participant = participantService.getParticipant(study, session.getId(), false);
         
-        return okResult(participant);
+        // Do not return healthCode. This will shortly be handled with Jackson filters.
+        StudyParticipant updated = new StudyParticipant.Builder().copyOf(participant)
+                .withHealthCode(null).withEncryptedHealthCode(null).build();
```

Comment:
This is a temporary "just in case", I'm going to replace this with Jackson filters which is a little cleaner.